bin/utils/node-utils.sh

```diff
@@ -149,10 +151,10 @@ check_node_version() {
   
   too_low=""
   too_low_support=""
-  if [[ ${node_major_version} -lt 8 ]]
+  if [[ ${node_major_version} -lt ${minVersion} ]]
   then
     too_low="true"
-  elif [[ ${node_major_version} -eq 8 ]] && [[ ${current_year} -gt 2021 ]]
+  elif [[ ${node_major_version} -eq ${minVersion} ]] && [[ ${current_year} -gt 2021 ]]
```

Comment:
this would mean you cant use v14 in the year 2023. you can.
just remove this whole elif section, its not useful anymore.